Jenkinsfile

```diff
@@ -57,9 +57,9 @@ pipeline {
                     // extract which specific storage provider to test
                     if (body.contains("[test cassandra]") || title.contains("cassandra:")) {
                         env.testProvider = "cassandra"
-                    } else if (body.contains("[test ceph]") || title.contains("ceph:")) {
-                        // For Ceph storage provider we are using GitHub actions to run test
-                        if (body.contains("[test full]")) {
+                    } else if (title.contains("ceph:")) {
+                        // Run the Ceph tests in Jenkins if specifically requested
+                        if (body.contains("[test ceph]")) {
```

Comment:
Why only if `ceph:` is in the title? What if I want to run tests for a PR where I change build settings?

Reply:
good point, will add that case...